react/features/base/conference/middleware.js

```diff
@@ -80,6 +81,9 @@ MiddlewareRegistry.register(store => next => action => {
     case DATA_CHANNEL_OPENED:
         return _syncReceiveVideoQuality(store, next, action);
 
+    case PARTICIPANT_UPDATED:
```

Comment:
Should this be in participants middleware?

Reply:
I was actually thinking about these a lot. But at the end I had 2 reasons to put it here:
1: actions triggered by a feature are usually (not always, of course) handled by the reducer of the feature itself, or the middleware of other features. Handling an action of a feature by the middleware of the feature itself seems odd for me, but it may be only my OCD.
2: We're modifying the conference object here, not the participant, so I thought it belongs to conference.

What do you think?

Reply:
I think number 2 makes the most convincing case to me, then participants becomes reactionary like it is for remote participants. It's off to me thought that the action type is from participants, but I think the circular nature that already exists means there will always be something a bit off to me.